app/models/search.rb

```diff
@@ -1,7 +1,7 @@
 class Search
 
   def execute type, term
-    puts "TYPE: '#{type}'"
+    puts "TYPE: '#{type}', term: #{term}"
```

Comment:
I think we should remove this...the app should be returning a good json error message with this data in it if it doesn't match one of the types.